js/filter.js

```diff
@@ -0,0 +1,28 @@
+'use strict';
+
+(function () {
+  var housingTypeFilter = document.querySelector('#housing-type');
+  var houseFiltration = function (pinsArr) {
+    if (housingTypeFilter.value !== 'any') {
+      var newPins = pinsArr.filter(function (pin) {
+        return pin.offer.type === housingTypeFilter.value;
+      });
+      return newPins;
+    } else {
+      return pinsArr;
```

Comment:
ветку else можно опустить и просто return
```suggestion
      return pinsArr;
```